.gitlab-ci-check-golang-unittests.yml

```diff
@@ -34,8 +34,8 @@ test:unit:
     GIT_DEPTH: 0
   before_script:
     # Install code coverage tooling
-    - go get -u github.com/axw/gocov/gocov
-    - go get -u golang.org/x/tools/cmd/cover
+    - GO111MODULE=off go get -u github.com/axw/gocov/gocov
+    - GO111MODULE=off go get -u golang.org/x/tools/cmd/cover
```

Comment:
I don't think these are required anyway, and we could have removed them. We are using the standard `go test -cover*`